tsconfig-strict.json

```diff
@@ -37,7 +37,6 @@
     "core/templates/domain/classifier/answer-classification-result.model.ts",
     "core/templates/domain/classifier/answer-classification-result.model.spec.ts",
     "core/templates/domain/classifier/classifier.model.ts",
-    "core/templates/domain/classifier/classifier.model.spec.ts",
```

Comment:
Why this needs to be removed?

Reply:
As discussed before, this is causing issue with ts strict compilation because classifier.model.spec.ts file imports text classifier protobuf file which is auto-generated but fails with ts compiler.

Reply:
Hmm, okay will only this file be affected?

Reply:
Yes